tests/distributions/test_continuous.py

```diff
@@ -284,6 +289,10 @@ def test_normal(self):
             {"mu": R, "sigma": Rplus},
             lambda q, mu, sigma: st.norm.ppf(q, mu, sigma),
         )
+        check_selfconsistency_icdf(
```

Comment:
We don't need to test this for scipy distributions that have ppf, I meant just for you to confirm it's working well

Reply:
Done.
I've removed these tests where scipy ppf functions are available.